src/Telemetry/Opt_In_Template.php

```diff
@@ -40,17 +59,26 @@ public function enqueue(): void {
 	protected function get_args() {
 
 		$optin_args = [
-			'plugin_logo'        => Admin_Resources::get_asset_path() . 'resources/images/stellar-logo.svg',
-			'plugin_logo_width'  => 151,
-			'plugin_logo_height' => 32,
-			'plugin_logo_alt'    => 'StellarWP Logo',
-			'plugin_name'        => 'The Events Calendar',
-			'user_name'          => wp_get_current_user()->display_name,
-			'permissions_url'    => '#',
-			'tos_url'            => '#',
-			'privacy_url'        => '#',
+			'plugin_logo'           => Admin_Resources::get_asset_path() . 'resources/images/stellar-logo.svg',
+			'plugin_logo_width'     => 151,
+			'plugin_logo_height'    => 32,
+			'plugin_logo_alt'       => 'StellarWP Logo',
+			'plugin_name'           => 'The Events Calendar',
+			'user_name'             => wp_get_current_user()->display_name,
+			'permissions_url'       => '#',
+			'tos_url'               => '#',
+			'privacy_url'           => '#',
+			'opted_in_plugins_text' => __( 'See which plugins you have opted in to tracking for', 'stellarwp-telemetry' ),
 		];
 
+		$optin_args['opted_in_plugins'] = array_map(
+			function( $plugin ) {
+				$plugin_data = get_plugin_data( WP_PLUGIN_DIR . '/' . $plugin . '/' . $plugin . '.php' );
+				return $plugin_data['Name'] ?? $plugin;
```

Comment:
🌮 Nice!

Reply:
will `$plugin` just return the plugin slug?

Reply:
Yes, $plugin_data['Name'] could be empty so as a fallback it could just return the slug